src/index.coffee

```diff
@@ -25,7 +25,7 @@ spawn = require('child_process').spawn
 path  = require 'path'
 
 JAVA_PATH = exports.JAVA_PATH = 'java'
-JAR_PATH  = exports.JAR_PATH  = path.join __dirname, 'node_modules/google-closure-compiler/compiler.jar'
+JAR_PATH  = exports.JAR_PATH  = path.join __dirname, '../node_modules/google-closure-compiler/compiler.jar'
```

Comment:
NB: this was apparently a leftover from previously, where an author changed the path in `index.js` (the generated file), but not here.